src/features/DebugSession.ts

```diff
@@ -187,9 +260,15 @@ export class DebugSessionFeature extends LanguageClientConsumer
             config.current_document = true;
         }
 
+        // Prevent the Debug Console from opening
+        config.internalConsoleOptions = "neverOpen";
+
+        const settings = getSettings();
+        config.createTemporaryIntegratedConsole ??= settings.debugging.createTemporaryIntegratedConsole;
+
         if (config.script === "${file}" || config.script === "${relativeFile}") {
             if (vscode.window.activeTextEditor === undefined) {
-                void this.logger.writeAndShowError("To debug the 'Current File', you must first open a PowerShell script file in the editor.");
+                await this.logger.writeAndShowError("To debug the 'Current File', you must first open a PowerShell script file in the editor.");
```

Comment:
```suggestion
                void this.logger.writeAndShowError("To debug the 'Current File', you must first open a PowerShell script file in the editor.");
```

So I know this is weird, but it's an ESLint thing. `await` here will actually halt the extension while the error pane is shown. Because of the (recommended and enforced) ESLint of `no-floating-promises` you can't just drop the `await`. However, we have that rule configured like so:

```
    "@typescript-eslint/no-floating-promises": [
      "error",
      {
        "ignoreVoid": true
      }
    ],
```

Such that `void` allows the async method to fire and is explicitly "we don't need to wait on it."

Reply:
I was wondering why the void was there, that makes sense.

Reply:
I think this fix got dropped in a rebase, should still be made `void` so the popup shows but doesn't block.

Reply:
Probably goofed, good catch. Fixed 2c6e818bd81643a935e34879aeec2baec9065640